### DISCUSSION TITLE
Full Translation Patch v0.0.242

### DISCUSSION THREAD

**Discussion by furiousg4m3r** [furiousg4m3r/Tales-of-Destiny-DC#1759] [Patch Release Discussion]
Changelog:
Linebreak fixes
Duplicate dialogue fix
Job Guy tweaks
Updated SLPS - Items, weapons, armor, tutorial, etc descriptions
Skit Editing: 100%
Menus: 100%
Scenario Files Translated: 100%
Editing: 100%

Credits and a few other small things still need completing.
**Continuing proofread fixes to Story, NPCs and Skits.
Disclaimer: There may be some Japanese left in some obscure places.
Please report any Japanese that is found.

Still a "Work In Progress". Save frequently, and post issues with screenshots of text on Github or Discord.

<hr /><em>This discussion was created from the release <a href='https://github.com/lifebottle/Tales-of-Destiny-DC/releases/tag/v0.0.242'>Full Translation Patch v0.0.242</a>.</em>

**Comment by Pugtaters** [comment]
This is the good patch to take if you encounter crashes on physical PS2. 243 and 244 were crashing for me, 242 doesn't.

**Reply by JulianoW** [reply]
244b2 shouldn't be crashing on hardware. If that's still the case you should post an issue with details so that we can investigate.

**Reply by Pugtaters** [reply]
243 and 244b2 were crashing for me the moment I went into Sound Mode and about 5-10 minutes of gameplay, the audio would stop playing and the game would crash. On an Slim NTSC 70001. This issue was happening with OPL and with a direct burned dvd.

**Reply by JulianoW** [reply]
Sound Test should be fixed in the next release.

As for the other crash, is there any additional information you can give? Is it random? In battles? Always at the same spot? Different spots?

Have you tried an unpatched version and does it crash?

We currently have no other reports of hw crashes on 244b2, and we've had a few people testing. So right now I have nothing on that.

**Reply by Pugtaters** [reply]
It happened to me in two different spots. The first time was in front of the crates "puzzle" on the ship, game just crashed. Second time was after awakening in the first town, Garr was there and suddenly there was no audio. No speech, no music. Then it eventually crashed.
